db/index.js

```diff
@@ -73,7 +76,7 @@ export const getPgPools = async () => {
 }
 
 /**
- * @param {pg.Client} client
+ * @param {pg.Client | pg.Pool} client
```

Comment:
Unfortunately, in node-pg typings describe Client and Pool in such way that one cannot be assigned to other 😢

Reply:
What about we add this type to this repo instead?

Reply:
Hang on! Maybe this will work - let me test it:
```ts
export type Queryable = Pick<Pool, 'query'>
```

Reply:
Works like a charm! See [0324c01](https://github.com/filecoin-station/spark-stats/pull/144/commits/0324c010c4f6817313c12f42b56a3b5e44eda1e9)